python/tvm/topi/adreno/utils.py

```diff
@@ -571,6 +572,39 @@ def get_texture_storage(shape):
         return "global.texture-weight"
 
 
+@register_func("tvm.info.mem.global.texture")
+def mem_info_global_texture():
+    return tvm.ir.make_node(
+        "MemoryInfo",
+        unit_bits=16,
+        max_num_bits=16384 * 16384 * 4 * 32,
+        max_simd_bits=4 * 32,
+        head_address=None,
+    )
```

Comment:
This looks to be repeated in each mem_info function so perhaps a common function called in each to prevent accidental changes in one and not the others.

Reply:
Good point, I moved this to the separate variables above .